src/Nest/DSL/UpdateSettingsDescriptor.cs

```diff
@@ -1,116 +1,154 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
 using Elasticsearch.Net;
 using Newtonsoft.Json;
 
 namespace Nest
-{
+{ 
+	public class SettingNames
```

Comment:
Love moving this out of the attributes :+1: suggestion on different name `UpdatableSettings`